pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.7.0</version>
+		<relativePath/>
+		<!-- lookup parent from repository -->
+	</parent>
+
+	<groupId>io.jenkins.pluginhealth.scoring</groupId>
+	<artifactId>plugin-health-scoring</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+
+	<name>Plugin Health Scoring</name>
+	<description>GSoC 2022 project</description>
+
+	<developers>
+		<developer>
+            <id>adi10hero</id>
+            <name>Aditya Srivastava</name>
+        </developer>
+		<developer>
+            <id>alecharp</id>
+            <name>Adrien Lecharpentier</name>
+        </developer>
+		<developer>
+            <id>dheerajodha</id>
+            <name>Dheeraj Singh Jodha</name>
+        </developer>
+		<developer>
+            <id>jleon33</id>
+            <name>Jake Leon</name>
+        </developer>
+    </developers>
+
+	<properties>
+		<java.version>17</java.version>
```

Comment:
I read about this and could understand that these additions would make UTF-8 a standard when it comes to encoding format. Because if it's not there, we might have some plugins (that process plain text files but have not been configured to use a specific file encoding) that would throw an error like: `[WARNING] Using platform encoding (Cp1252 actually) to copy filtered resources, i.e. build is platform dependent!`

Please correct me and/or let me know if I should know anything more about this, thank you!!